tests/acceptance/environment.py

```diff
@@ -24,5 +24,5 @@ def before_scenario(context, scenario):
         context.multi_db = True
 
 
-def django_ready(context):
+def django_ready(context, scenario):
```

Comment:
If possible the additional parameter should be optional.

Maybe change the documentation instead, or in addition?

Reply:
This is a strange situation to be in for sure. Either leave the functionality as is and change the docs, which would be a shame in my opinion. Given that there may be useful information in the scenario where you want to create data states across a number of features.

Whereas if you add this are it'll break existing projects raising `IndexError`. I'm open to suggestions on how to make it optional. But like I said, I'm just looking into adding this package into a project. So I'm fairly new to the behave source.

Perhaps a new cli flag could do the trick where the default remains. Not sure though, it's up to you all.

Reply:
Maybe change [this](https://github.com/behave/behave-django/blob/master/behave_django/management/commands/behave.py#L148) to be something like `monkey_patch_behave(django_test_runner, options['django_hook_add_scenario'])`

Reply:
This will always be `None` though wouldn't it? By passing `*args` you're not really solving anything, why not just leave it as is, and change the docs to say `def django_ready(context):` ?

Reply:
@anthonyalmarza I think @kingbuzzman is correct. Would you mind to just fix the docs that way?